debian/copyright

```diff
@@ -1,5 +1,38 @@
-Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: revpi-repo
+Upstream-Contact: <preferred name and address to reach the upstream project>
+Source: <url://example.com>
 
 Files: *
-Copyright: 2016-2017 Kunbus GmbH
-License: GPL-2.0
+Copyright: 2021 Frank Pavlic <f.pavlic@kunbus.com>
```

Comment:
How is it that Frank is now the copyright holder? And why has the Kunbus GmbH lost its copyright from 2016?

Reply:
Oh, sorry, Frank is the Maintainer,  I made a mistake. So it should be still " 2016-2021 Kunbus GmbH" ?

Reply:
I guess so